scripts/e2e

```diff
@@ -5,10 +5,28 @@ source $(dirname $0)/../scripts/lib/debug_functions
 
 cd $(dirname $0)
 
+echo $1
```

Comment:
These echos shall be removed. It was added for debugging.

Reply:
Done